expertsystem/nested_dicts.py

```diff
@@ -309,3 +321,85 @@ def _particle_with_spin_projection_to_dict(
         if item["Type"] == "Spin":
             item["Projection"] = spin_projection
     return output
+
+
+def _convert_nodes_to_dict(
```

Comment:
It doesn't really matter while this function is temporary (right?), but `_convert_nodes_to_dict` modifies its input, while `_convert_edges_to_dict` creates a new object and returns it.